packages/mobile/src/tokens/saga.test.ts

```diff
@@ -140,3 +142,52 @@ describe(tokenAmountInSmallestUnit, () => {
     ).rejects.toThrowError(`Couldnt find token info for address ${mockAddress}.`)
   })
 })
+
+describe('watchAccountFundedOrLiquidated', () => {
+  beforeEach(() => {
+    // https://github.com/jfairbank/redux-saga-test-plan/issues/121
+    jest.useRealTimers()
+    jest.clearAllMocks()
+  })
+
+  const balances = (firstValue: BigNumber | null, restValue: BigNumber | null) => {
+    let callCount = 0
+    return () => {
+      callCount += 1
+
+      switch (callCount) {
+        case 2:
+        case 3:
+          return restValue
+        default:
+          return firstValue
+      }
+    }
+  }
+
+  it('dispatches the account funded event', async () => {
```

Comment:
Should we add a test that going from `null` to something non zero doesn't track `account_funded` because it's an account restore?

Reply:
yeah good idea, i added a test case for this :)